assembl/static2/js/app/actions/authenticationActions.js

```diff
@@ -15,7 +15,7 @@ const signupGeneralError = error => ({
 export const signupAction = payload => dispatch =>
   signUp(payload)
     .then(() => {
-      const relevantCookies = COOKIE_TYPES.filter(c => c === 'ACCEPT_CGU' || c === 'ACCEPT_PRIVACY_POLICY_ON_DISCUSSION');
+      const relevantCookies = TO_ACCEPT.includes(c => c);
```

Comment:
TO_ACCEPT.includes(c) not c=>c (is a identity function)
TO_ACCEPT change this name

Reply:
COOKIE_TYPES.filter(c => TO_ACCEPT.includes(c))